### ISSUE TITLE
A few puzzle about training.

### ISSUE THREAD

**Issue by TopGun666** [JiawangBian/SC-SfMLearner-Release#27]
Hi Jiawang @JiawangBian 

I have a few puzzle about training.

1. I try set the sequence length at 6 (instead of 3), however its smooth loss and consistency loss decrease at first 10 epochs and then then keep increasing. Maybe It is overfitting? What's the meaning of the sequence length? What is its difference from batch size?

2. Have you try to use other batch size? Or in other words, 4 is the best batch size you have tried.

3. As we know, relative pose is from source image to target image plane. That means every 4 (batch size) images we can only get 3 relative pose. However your PoseNet outputs 4 poses every batch. I find you don't use the poes to warp the target image to ref_image in your code. It seems your PoseNet output is not the relative pose from source image to target image but it seems like the pose from World frame to camera frame. The same situation happen on test, which every image input could get a pose. 

Best regards,

Yu

**Comment by JiawangBian** [comment]
Sequence length = 3 means that the the second frame is tartget image, and the others are source images. So it will output two relative camera poses (from the frame 2 to frame 1, and to frame 3). This is one traning example. The batch size refers to how many examples you train togather at the same time.

**Comment by TopGun666** [comment]
Thanks a lot. What you said is consistent with the results I verified. What a pity we couldn't see your live broadcast last night.

**Comment by JiawangBian** [comment]
So sorry for that. Finally I find that the reason is that the Austrlian IP address are blocked by Chinese firewall... I will upload a video today.
